scripts/verify-phase-52-1-cli-command-contract.sh

```
@@ -0,0 +1,154 @@
+    exit 1
+  fi
+done
+
+for field in "${shared_output_fields[@]}"; do
+  if ! grep -Eq -- "^- \`${field}\`: [^[:space:]].+$" "${doc_path}"; then
+    echo "Missing or empty Phase 52.1 shared output field: ${field}" >&2
+    exit 1
+  fi
+done
+
+if ! grep -Fxq -- '- `result`: one of `ok`, `skipped`, `mocked`, `degraded`, or `failed`.' "${doc_path}"; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)